tools/run_tests/run_tests.py

```diff
@@ -444,14 +446,21 @@ def _build_and_run(check_cancelled, newline_on_success, travis, cache):
                     newline_on_success=newline_on_success, travis=travis):
     return 1
 
-  # run all the tests
-  all_runs = itertools.chain.from_iterable(
-      itertools.repeat(one_run, runs_per_test))
-  if not jobset.run(all_runs, check_cancelled,
-                    newline_on_success=newline_on_success, travis=travis,
-                    maxjobs=min(args.jobs, min(c.maxjobs for c in run_configs)),
-                    cache=cache):
-    return 2
+  # start antagonists
+  antagonists = [subprocess.Popen(['tools/run_tests/antagonist.py']) 
+                 for i in range(0, args.antagonists)]
```

Comment:
Since you make no use of i, don't give it a name; just use underscore:

for _ in range(0, args.antagonists)